modules/packages/LinearAlgebra.chpl

```diff
@@ -2605,16 +2611,16 @@ module Sparse {
   private proc sortIndices(ref A: [?Dom] ?eltType) where isCSArr(A) {
     use Sort;
 
-    const (M, N) = A.shape;
+    const row_range = Dom.dim(0);
```

Comment:
Chapel typically uses camelCase:

```suggestion
    const rowRange = Dom.dim(0);
```

Reply:
Made this change on the sparse-0based branch.